tools/rostopic/src/rostopic/__init__.py

```diff
@@ -1218,6 +1318,23 @@ def eval_fn(m):
         filter_expr = None
     _rostopic_hz(topic, window_size=window_size, filter_expr=filter_expr)
 
+
+def _rostopic_cmd_delay(argv):
+    args = argv[2:]
+    import argparse
```

Comment:
Please move the imports to the top of the module (in alphabetical order following PEP8).